frontends/ol-components/src/components/LearningResourceCard/LearningResourceCard.stories.tsx

```diff
@@ -28,9 +28,10 @@ const meta: Meta<typeof LearningResourceCard> = {
       options: ["Loading", "Without Image", ...Object.values(ResourceTypeEnum)],
       mapping: {
         Loading: undefined,
-        "Without Image": makeResource({
+        "Without Image": {
+          ...makeResource(),
           image: null,
```

Comment:
This change is unrelated, but was something I did while opening 
- https://github.com/mitodl/hq/issues/4774

Decided to keep it.